cmd/buildah/config.go

```diff
@@ -17,99 +17,71 @@ const (
 )
 
 var (
-	configurationFlags = []cli.Flag{
+	configFlags = []cli.Flag{
+		cli.StringFlag{
+			Name:  "name",
+			Usage: "name or ID of the working container",
+		},
+		cli.StringFlag{
+			Name:  "root",
+			Usage: "root directory of the working container",
+		},
+		cli.StringFlag{
+			Name:  "link",
+			Usage: "symlink to the root directory of the working container",
+		},
```

Comment:
Above the verbiage was "`pathname` of a symlink to the root directory of the working container" .  Should it be the same here?